scylla/src/transport/session_builder.rs

```diff
@@ -446,13 +446,13 @@ mod tests {
     #[test]
```

Comment:
(I can't attach it to the correct line in this file, so I'm placing this comment here)

You need to update the docstring comment for `SessionBuilder::tcp_nodelay` so that it says that the default is true (line 146).